dev/app/app.js

```diff
@@ -1,6 +1,11 @@
 'use strict';
 
 /**
+ * Change to switch between local JSON and firebase
+ */
+var localData = true;
+
```

Comment:
I think it would be better give all scenarios to test. It's safer and you will be amazed what people can do : break the unbreakable. Personally I prefer have all the variable scenario outside `gulp`. Just have a file as before that they can't commit but they can change it to test. I wouldn't have more than two test with gulp: one for `dist` and one for `dev`. I think it would be cleaner.